02Lesson/01Project/index.html

```diff
@@ -1,9 +1,61 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>My Bio</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+   <section>
+    <h1>
+      Travis Schlicke
+    </h1>
+    <p>
+      My address goes here
+    </p>
+  </section>
+    <h3>
+      Bio
+    </h3>
+    <p>
+      I am a man that believes to never stop learning. From learning how to give masssages to learning how to cook.
+      I have many things I have learned. My latest adventure is taking this computer class to learn how to code.
+      I am a man that is classified as a nerd and I love that I am. I am huge into video games and sports. Sports
+      is not a typical thing nerds enjoy but for me I love them. My favorites are Soccer and American Football. Go Cowboys.
+    </p>
+  <hr>
```

Comment:
Indentation: See above, after closing tag, next opening tag needs to be on same level.